dotcom-rendering/src/components/Timeline.stories.tsx

```diff
@@ -20,7 +21,14 @@ const format: ArticleFormat = {
 const meta = {
 	component: Timeline,
 	title: 'Components/Timeline',
-	decorators: [splitTheme([format], { orientation: 'vertical' })],
+	parameters: {
+		chromatic: {
+			modes: {
+				vertical: allModes.sideBySideVertical,
```

Comment:
Isn't this called `splitVertical`?

Reply:
Do you mean the `vertical` mode? The naming here doesn't matter. I went with vertical as [we've used horizontal elsewhere](https://github.com/guardian/dotcom-rendering/blob/3089bafc53008694e6ec73fdd7efd04f01234b0c/dotcom-rendering/src/components/QAndAExplainers.stories.tsx#L69)